torch/package/package_exporter.py

```diff
@@ -231,8 +231,13 @@ def __init__(
         if isinstance(f, (Path, str)):
             f = str(f)
```

Comment:
```suggestion
            f = os.fspath(f)
```
is what it should be btw